cypress/unit/store-mutation-createColumnToCategoryMap.cy.js

```diff
@@ -0,0 +1,38 @@
+import { getters, mutations } from "~/store/index-refactor";
+
+var store = {};
```

Comment:
```suggestion
let store = {};
```
to be consistent with the rest of the codebase